app/Database/migrations/2024_12_26_014452_default_theme_setting_change.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
```

Comment:
I don't think you need to do this - just set it to "seven" or whatever for new installs. This is also likely only true for new installs, anyway.

Reply:
Yeah, I guess we could just edit the `settings.yml`